hhs_hosp/delphi_hhs/run.py

```diff
@@ -77,6 +77,7 @@ def run_module(params):
             - "log_filename" (optional): str, name of file to write logs
             - "epidata" (optional): dict, extra parameters to send to Epidata.covid_hosp
     """
+    Epidata.auth = ('epidata', str(params["validation"]["common"]["api_credentials"]))
```

Comment:
Should probably make this fail-safe, in case these keys don't exist
```suggestion
    if params.get("validation") and params["validation"].get("common") and params["validation"]["common"].get("api_credentials"):
        Epidata.auth = ('epidata', str(params["validation"]["common"]["api_credentials"]))
```